packages/logger/tests/unit/middleware/middy.test.ts

```diff
@@ -10,6 +10,7 @@ import { Logger } from './../../../src';
 import middy from '@middy/core';
 import { PowertoolLogFormatter } from '../../../src/formatter';
 import { Console } from 'console';
+import { context as dummyContext } from '../../../../../tests/resources/contexts/hello-world';
```

Comment:
I think that unless you're doing something specific with the context in each test case you could use this one instead of redeclaring it in each test, alternatively just remove the line.